### DISCUSSION TITLE
Cannot find the `mathjs/lib` folder

### DISCUSSION THREAD

**Discussion by redbar0n** [josdejong/mathjs#2650] [Q&A] [answered]
[mathjs/main/esm/index.js](https://github.com/josdejong/mathjs/blob/develop/main/esm/index.js):
```js
// TODO: deprecated since v8, remove this deprecation warning in v9
throw new Error('The file "mathjs/main/esm/index.js" has been moved since mathjs@8.0.0. ' +
  'Please load "mathjs" or "mathjs/lib/esm/index.js" instead.')
```

But there is no `mathjs/lib/` folder...?

**Comment by josdejong** [comment, marked as answer]
Here is the folder (in the npm library, it does not exist in the source code): https://unpkg.com/browse/mathjs@10.4.0/lib/
